mava/systems/tf/variable_sources.py

```diff
@@ -0,0 +1,123 @@
+import os
+import time
+from typing import Any, Dict, Sequence, Union
+
+import numpy as np
+from acme.tf import utils as tf2_utils
+
+from mava.systems.tf import savers as tf2_savers
+
+
+class VariableSource:
```

Comment:
Is this also adapted from acme?

Reply:
No this is a new Mava variable_source that stores all the variables in a centralised place. This allows the executors, evaluators and trainers to set and get the latest variables.